prow/config/config.yaml

```diff
@@ -129,6 +129,63 @@ branch-protection:
                   - "unit-test"
                   - "license/cla"
                 strict: true
+        tidb:
+          branches:
+            master:
+              enforce_admins: false
```

Comment:
can this config forbid admin to merge PRs when the required checks are failed?

Reply:
Yes, just change the value to true. Do we want to force it on admins as well?

Reply:
Enabled.